libs/pages/application/src/lib/feature/page-settings-general-feature/page-settings-general-feature.tsx

```diff
@@ -8,6 +8,12 @@ import { ApplicationEntity } from '@console/shared/interfaces'
 import { AppDispatch, RootState } from '@console/store/data'
 import PageSettingsGeneral from '../../ui/page-settings-general/page-settings-general'
 
+export const buildGitRepoUrl = (provider: string, branch: string): string => {
```

Comment:
could be nice to test this kind of very small utility functions

Reply:
Yes, I already tested it you can see it in the file `page-settings-general-feature.spec.tsx` !